gremlin-client/src/io/mod.rs

```diff
@@ -59,10 +59,10 @@ impl GraphSON {
                 "@type" : "g:UUID",
                 "@value" : s.to_string()
             })),
-            (GraphSON::V1, GValue::Date(d)) => Ok(json!(d.timestamp())),
+            (GraphSON::V1, GValue::Date(d)) => Ok(json!(d.timestamp_millis())),
```

Comment:
Touching the GraphSON V1 implementation is something I flip-flopped on since as far as I can tell in the [docs] (http://tinkerpop.apache.org/docs/current/dev/io/#graphson-1d0) there is no spec definition for a date, it's just some integer value, which seems consistent with the test I ended up modifying in gremlin-client/tests/integration_client_v1.rs.

In that test the value is parsed out as a `i64` so I think there's an argument to be made that there is no millisecond precision expectation in the v1 standard so it could be left as is as epoch second but for now I went for consistency with v2 and v3. Let me know if you feel like I should have left it alone @wolf4ood